src/env.cc

```diff
@@ -1473,4 +1494,8 @@ Local<FunctionTemplate> BaseObject::GetConstructorTemplate(Environment* env) {
   return tmpl;
 }
 
+bool BaseObject::IsAllowedStrongObjectAtExit() const {
+  return IsWeakOrDetached();
+}
```

Comment:
Part of me wants to suggest `IsNotIndicativeOfMemoryLeakAtExit()` or `IsGarbageCollectibleAfterExit()`, but most of me wishes that part of me would stop suggesting terrible names for things.

Reply:
`IsNotIndicativeOfMemoryLeakAtExit()` sounds fine to me, I’ll update :woman_shrugging: